lib/omc/stack_command.rb

```diff
@@ -100,8 +101,11 @@ def bastion
 
     def default_ssh_args
       [].tap do |args|
-        args.push("-o", "ProxyCommand ssh -W %h:%p #{bastion.host}") if bastion
+        proxy_command = "ProxyCommand ssh -W %h:%p #{bastion.host}"
+        proxy_command += " -R 3101:localhost:3000" if @remote_forward
+        args.push("-o", proxy_command) if bastion
         args.push("-A") if @forward_agent
+        args.push("-R 3101:localhost:3000") if @remote_forward
```

Comment:
Do we want to allow any configuration of these port numbers? Seems like `remote_forward` would be fairly limited in use without it. If you have a limited use case in mind, perhaps a different variable name and more specific commit message on https://github.com/StemboltHQ/omc/pull/19/commits/54d039f10c2734594f54b70f4ce3360e79bd2d93 would be appropriate.